Adjust shebang for split /usr
Kill possible /tmp vulnerability on service restart
--- stalld-1.9.0/scripts/throttlectl.sh.orig	2021-02-17 16:10:23.000000000 +0100
+++ stalld-1.9.0/scripts/throttlectl.sh	2021-02-22 17:58:48.425800258 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 
 # This script is called to either turn off or turn on RT throttling
 # The 'off' argument causes the current values of the throttling
@@ -9,7 +9,7 @@
 
 path=/proc/sys/kernel
 cmd=$1
-savefile=/tmp/rtthrottle
+savefile=/var/run/rtthrottle
 defperiod=1000000
 defruntime=950000
 case ${cmd} in
